build/static/style.css

```diff
@@ -0,0 +1,21 @@
+.workload-summary-cell{
+  max-width: 40px;
+  overflow: hidden;
+}
+
+.red, .green{
+  color: white;
+}
```

Comment:
also what the heck is this?

Reply:
Making sure text on a red or green field is white for contrast. About to no longer be an issue as Olena wants me to use checkmarks and x's instead of big color fields.